app/assets/javascripts/discourse/app/form-kit/components/fk/form.gjs

```diff
@@ -22,7 +23,9 @@ import FKSection from "discourse/form-kit/components/fk/section";
 import FKSubmit from "discourse/form-kit/components/fk/submit";
 import { VALIDATION_TYPES } from "discourse/form-kit/lib/constants";
 import FKFormData from "discourse/form-kit/lib/fk-form-data";
+import { headerOffset } from "discourse/lib/offset-calculator";
 import { i18n } from "discourse-i18n";
+import { getScrollParent } from "float-kit/lib/get-scroll-parent";
```

Comment:
Not ideal to import something from float-kit, but this is easy to move around if necessary.